server/webapp/WEB-INF/rails/webpack/views/components/forms/input_fields.tsx

```diff
@@ -36,7 +36,7 @@ export interface RequiredFieldAttr {
 }
 
 interface LabelAttr extends RequiredFieldAttr {
-  label?: string;
+  label?: m.Children;
```

Comment:
We should be able to embed tooltip components and such in our labels. This should not break any existing code as `string` satisfies `m.Children`